demo_scenario.py

```diff
@@ -31,22 +31,26 @@ def set_up_scenario(
                 "V_cap"],
             digital_signals = [])
 
+        CAPTURE_DURATION = 0.1
+
         # Create and schedule switch close and open events
         close_event = typhoon_automator.Utility.create_callback_event(
             message = "Closing switch",
             callback = DemoScenario.close_switch)
-        close_time = random.uniform(0.0, self._duration / 2)
+        close_time = random.uniform(CAPTURE_DURATION / 2, self._duration / 2)
```

Comment:
this makes it so that at least some of the scenario plays out, right?

Reply:
before the close event**

Reply:
There was a problem with random values scheduling things at invalid times.  I forget the exact reason for delaying the close time, but we'll leave this as-is for now (the example will be updated later)